src/libexpr/primops.cc

```diff
@@ -2470,8 +2469,8 @@ static void prim_catAttrs(EvalState & state, const PosIdx pos, Value * * args, V
     auto attrName = state.symbols.create(state.forceStringNoCtx(*args[0], pos));
     state.forceList(*args[1], pos);
 
-    Value * res[args[1]->listSize()];
-    unsigned int found = 0;
+    boost::container::small_vector<Value *, 256> res(args[1]->listSize());
```

Comment:
Same with this 256?

Reply:
This is properly arbitrary. I think it's safe to assume that these numbers are guesswork, whether that's actual guesswork or an outdated benchmark.

Reply:
I did name and document this after all. See `nonRecursiveStackReservation`